packages/cx-layout/src/components/WhereToBuy/WhereToBuyClean.tsx

```diff
@@ -0,0 +1,47 @@
+/**
```

Comment:
Is there a reason we don't just use the existing link component for this and/or create a basic button component?